packages/lodestar/README.md

```diff
@@ -3,42 +3,51 @@
 [![Discord](https://img.shields.io/discord/593655374469660673.svg?label=Discord&logo=discord)](https://discord.gg/aMxzVcr)
 ![ETH2.0_Spec_Version 0.9.4](https://img.shields.io/badge/ETH2.0_Spec_Version-0.9.4-2e86c1.svg)
 
-Typescript imlementation of the Eth2.0 beacon chain
 
-## What you need
-You will need to go over the [specification](https://github.com/ethereum/lodestar-specs). You will also need to have a [basic understanding of sharding](https://github.com/ethereum/wiki/wiki/Sharding-FAQs). Note that that the specification is an ongoing document and will get outdated. The reference implementation by the Ethereum development team is written in Python and can be found [here](https://github.com/ethereum/beacon_chain).
+## Prerequisites
 
-## Run dev
+* [Lerna](https://github.com/lerna/lerna)
+* [Yarn](https://yarnpkg.com/)
 
-` ./bin/lodestar dev -v 8 -p minimal`
 
-#### Using start-multi.sh
 
-`sh ./start-multi.sh <number_of_beacon_nodes> <total_validators> <validator_distribution_per_node>`
-`sh ./start-multi.sh 4 8 8,0,0,0`
+## What you need
+You will need to go over the [specification](https://github.com/ethereum/lodestar-specs). You will also need to have a [basic understanding of sharding](https://github.com/ethereum/wiki/wiki/Sharding-FAQs). Note that that the specification is an ongoing document and will get outdated. The reference implementation by the Ethereum development team is written in Python and can be found [here](https://github.com/ethereum/beacon_chain).
 
-#### Generating flame graphs from spec tests
-Example of cpu-profile generation of blocksanity tests in current directory
+## Getting Started
 
-`GEN_PROFILE_DIR=$PWD mocha -r ./.babel-register 'test/spec/sanity/blocks/blocksanity_s_mainnet.test.ts'`
+To get an instance of Lodestar up & running, start a terminal from the root of the this repo:
 
-To view results:
-```bash
-npm i -g 0x
-0x --visualize-cpu-profile ./0\ -\ blocksanity-1561705313935.cpuprofile
+1. Install dependencies accross all packages:
+```
+yarn install
 ```
-Open resulting html file in favorite browser.
 
-## Usage
-1. If you haven't `yarn install`
-2. `lerna run build`
-3. `packages/lodestar/./bin/lodestar --help`
+2. Run the build script:
+```
+yarn run build
+```
+
+3. Lodestar should now be ready for use:
+```
+yarn run cli --help
+```
 
 Note:
 
 Spec test cases are optional dependency which can be skipped by adding `--ignore-optional` when installing dependencies.
 You can always download spec test cases by running `yarn install --force`.
 
+## Run dev
+
+` ./bin/lodestar dev -v 8 -p minimal`
+
+#### Using start-multi.sh
+
+`sh ./start-multi.sh <number_of_beacon_nodes> <total_validators> <validator_distribution_per_node>`
+`sh ./start-multi.sh 4 8 8,0,0,0`
```

Comment:
I don't think this works anymore. I think I've commented it out on master

Reply:
Ok cool will comment out, although is there something to the same effect to put in?

Reply:
Not currently